modules/Makefile.kubernetes

```diff
@@ -71,6 +72,15 @@ KUBECTL_CMD ?= $(KUBECTL_SSH_CMD) "$(KUBECTL)" --logtostderr=true --insecure-ski
 # Reference Docs:
 #  - https://cloud.google.com/container-engine/docs/kubectl/
 #
+## Bring up SSH tunnel for kubernetes commands, must be called before executing any targets
+kubernetes\:tunnel-up:
```

Comment:
This would still force user to invoke `kubernetes:tunnel-down`. I think it's OK to force caller to ensure the tunnel is up. If they don't do so, they'll just get prompted for MFA multiple times :)